builder/files/heimdalld.service

```diff
@@ -8,10 +8,9 @@
   WorkingDirectory=/usr/local/bin
   ExecStart=/usr/local/bin/heimdalld start --home /var/lib/heimdall \
     --chain=mainnet \
-    --bridge --all \
     --rest-server
   Type=simple
-  User=ubuntu
   LimitNOFILE=65536
+  User=heimdall
```

Comment:
Do we need 2 users? Could we stick with just 1 user name? Like Bor?

Reply:
I think it is better to create two users, one for heimdall and one for bor. Using one user may be confusing. Also, one of the reasons to create new users is to get better file permission control.

Reply:
Ok, in nearest future I'm planning to merge Heimdall and Bor, then will remove one of this users.